src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/template.json

```
@@ -254,6 +254,48 @@
         }
       ]
     },
+    "XUnitVersion": {
+      "type": "parameter",
+      "description": "The version of xUnit.net to use.",
+      "displayName": "xUnit.net version",
+      "datatype": "choice",
+      "isEnabled": "TestFramework == xUnit.net",
+      "replaces": "XUnitVersion",
+      "choices": [
+        {
+          "choice": "v2",
+          "displayName": "v2",
+          "description": "Chooses xUnit v2 with VSTest"
```

Always refer to it by its full name "xUnit.net"
```suggestion
          "description": "Chooses xUnit.net v2 with VSTest"
```